inc/commondbtm.class.php

```diff
@@ -5167,7 +5167,7 @@ static function generateLinkContents($link, CommonDBTM $item) {
     *                        - content_field (default content) the field who receive the main text
     *                                                          (with images)
     *                        - name (default filename) name of the HTML input containing files
-    *
+    *                        - date  Date to set on document_items
```

Comment:
```suggestion
    *
```

Reply:
Why do you want to remove this phpdoc line ?

Reply:
Sounds like I've read too fast; this is not to remove; my bad